.github/workflows/check-nixf-tidy.yml

```
@@ -0,0 +1,97 @@
+name: Check changed Nix files with nixf-tidy (experimental)
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, edited]
+permissions:
+  contents: read
+
+jobs:
+  nixos:
+    runs-on: ubuntu-latest
+    # if: "github.repository_owner == 'NixOS' && !contains(github.event.pull_request.title, '[skip treewide]')"
```

Honestly I don't get why some workflows use `github.repository_owner == 'NixOS'` at all. It's really annoying when wanting to test the workflow, as you've discovered.

Notably I didn't add this to the [`pkgs/by-name` check workflow](https://github.com/NixOS/nixpkgs/blob/master/.github/workflows/check-by-name.yml), and didn't hear anybody complain about it, so let's do the same here

```suggestion
    if: "!contains(github.event.pull_request.title, '[skip treewide]')"
```